components/ai_chat/resources/untrusted_conversation_frame/components/context_actions_assistant/index.tsx

```diff
@@ -80,6 +99,16 @@ export default function ContextActionsAssistant(
           })}
         />
       </Button>
+      {conversationContext.isLeoModel && props.turnModelKey &&(
```

Comment:
note: We don't render this component unless `conversationContext.isLeoModel` is true so we probably don't need to check it here.

Reply:
https://github.com/brave/brave-core/pull/28999/commits/4775c60d98dbf436e75a156f6ab82534c2a27151